cmake/draco.cmake

```diff
@@ -0,0 +1,20 @@
+#
+option(WITH_DRACO "Choose if Draco support should be built" TRUE)
+find_package(Draco QUIET 1.3.6)
+
+if (WITH_DRACO)
+    set_package_properties(Draco PROPERTIES TYPE RECOMMENDED
+        PURPOSE "Provides Draco compression")
+    if(Draco_FOUND)
+        include_directories(${draco_INCLUDE_DIR})
+        set(CMAKE_REQUIRED_LIBRARIES ${CMAKE_REQUIRED_LIBRARIES}
```

Comment:
I'm not sure we should be doing this.

Reply:
Why not? I didn't write this part, but it was working when I picked it up so I didn't mess with it.

Reply:
It throws a cmake warning on newer cmake setups.